src/views/Metadata.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div>
+    <div class="editor-back">
+      <router-link to="/">
+        <a>
+          <i class="fas fa-arrow-circle-left" /> Start
+        </a>
+      </router-link>
+    </div>
+    <h2>Zusätzliche Metadaten festlegen</h2>
+
+    <BaseChooser
+      v-if="displayConstraintChooser"
+      :object-to-edit="constraint"
+      :name="'Einschränkung'"
+      :operand-list="opList"
+      :operand-mapping="opMapping"
+      @chosen="addConstraint($event)"
+      @abort="hideConstraintChooser()"
+    />
+
+    <BaseButton class="add-constraint" type="button" @click="showConstraintChooser()">Einschränkung hinzufügen</BaseButton>
+
+    <li v-for="(constraint, index) in constraints" :key="index" class="constraints">
+      {{ description(constraint) }}
+      <BaseButton class="remove-constraint" remove @click="removeConstraint(index)">
+        <i class="fas fa-times" />
+      </BaseButton>
+    </li>
+  </div>
+</template>
+
+<script>
+import BaseChooser from '../components/BaseChooser.vue';
+import BaseButton from '../components/BaseButton.vue';
+import {
+  operandList,
+  operandMapping,
+  operatorList,
+} from '../libs/odrl/constraints';
+
+export default {
+  name: 'Metadata',
+  components: {
+    BaseButton,
+    BaseChooser,
+  },
+  data() {
+    return {
+      displayConstraintChooser: false,
+      constraint: null,
+      constraints: [],
+    };
+  },
+  computed: {
+    opList() {
+      const filteredOperands = operandList.filter((value, index, arr) => value != 'Nutzungsdauer' && value != 'Nutzeranzahl');
```

Comment:
Not every operand that we want in rule constraints makes sense in the global context.